src/GeositeFramework/plugins/launchpad/main.js

```diff
@@ -54,6 +54,18 @@ define([
                         var saveCode = $(e.target).data('saveCode');
                         self.plugin.turnOff();
                         self.triggerEvent('launchpad:activate-scenario', saveCode);
+                    })
+                    .on('click', 'button.truncate-toggle', function (e) {
+                        var parentEl = $(e.target.parentElement);
+                        parentEl.find('p').toggleClass('truncate-box');
+                        parentEl.find('.truncate-toggle').each(function () {
+                            if ($(this).attr('data-i18n') === 'See More') {
+                                $(this).attr('data-i18n', 'See Less');
+                            } else {
+                                $(this).attr('data-i18n', 'See More');
+                            }
+                            $(this).localize();
```

Comment:
Ultimately, we shouldn't be making decisions based on i18n attributes. We can pair on this.